pkg/sql/colexec/agg/bit_xor/bit_xor.go

```diff
@@ -23,11 +23,11 @@ func New[T1 types.Ints | types.UInts | types.Floats]() *BitXor[T1] {
 func (bx *BitXor[T1]) Grows(_ int) {
```

Comment:
Still troubles me, what do you mean by and/or/xor two floating point number?

Reply:
float will be cast to uint64